editor/scene/sprite_frames_editor_plugin.cpp

```diff
@@ -2654,7 +2655,9 @@ SpriteFramesEditor::SpriteFramesEditor() {
 	file_split_sheet->set_title(TTRC("Create Frames from Sprite Sheet"));
 	file_split_sheet->set_file_mode(EditorFileDialog::FILE_MODE_OPEN_FILE);
 	add_child(file_split_sheet);
-	file_split_sheet->connect("file_selected", callable_mp(this, &SpriteFramesEditor::_prepare_sprite_sheet));
+	// Deferred so file dialog is hidden when sprite sheet dialog popups. Otherwise, after allowing
+	// sprite sheet dialog to be maximized, it would complain about already having exclusive child window.
+	file_split_sheet->connect("file_selected", callable_mp(this, &SpriteFramesEditor::_prepare_sprite_sheet), CONNECT_DEFERRED);
```

Comment:
This feels hackish, but otherwise it would yell after I've added `split_sheet_dialog->set_flag(Window::FLAG_MAXIMIZE_DISABLED, false);`.

@bruvzg Is this expected? If yes, is there a better way to handle that than deferring?

Reply:
Depends on the way it's shown, if it's using autoparenting, than it will try to attach to still visible file dialog, so it is expected and deferred call is what is used in other cases.

Reply:
Not sure how it's related to maximize flag, probably just timing.